api/src/paths/user/{userId}/get-projects.ts

```diff
@@ -0,0 +1,145 @@
+import { RequestHandler } from 'express';
```

Comment:
Similar to the previous endpoint, lets change this to something like:
`paths/user/{userId}/projects/get`

Alternatively, it could also just be:
`paths/user/{userId}/get`
And in this case, it could be a general `get` for that user, and could return a whole packet of info about them (user details, projects they are one, etc).

Though that being said, I don't know if we have any need for the second option yet, so it might be premature to implement something like that now.